### DISCUSSION TITLE
multi forms saved on the same DB line

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2054] [Help]
Hi is there a way to get forms on multiple pages to fill the same DB line? 
for example,
I have a form my shift supervisor needs to fill out each day it is much too long for everything to fit on one page. I would love for the flow to be, submit one section of the form that would navigate to a new page and fill out / submit the next section of the form and so on until the form is complete. however, I don't want the data thrown about on multiple DB lines I need to keep it all on one line is this possible?

**Comment by mjashanks** [comment]
Hey Andrew. There is no good way to do this at the moment. You can do this, but its a bit of a hack (video below):

1. Create a blank record up front, using a create button inside a form. I have defaulted the only required field to "(new)" just so I can get a blank record created
2. Have a button navigate to your first screen
3. Your first screen has a button which saves and navigates to the second screen
4. etc etc, until you are finished, whereby you can return to the list screen

Drawback of this
1. You create a record up-front, meaning that you could be left with a blank record
2. Validation is broken as you are not allowed to save a record on the first screen if you have required fields on the second screen.

We would like to do this properly at some stage, as mentioned in https://github.com/Budibase/budibase/discussions/1062


https://user-images.githubusercontent.com/3524181/126547601-1e1e6e16-6172-4250-af74-ef92a12095e1.mp4

**Reply by KernOil** [reply]
this will work for my application, thank you very much!

**Reply by Bogey** [reply]
A tab component (or accordion) would be great for this. Are these on the roadmap?

**Reply by mjashanks** [reply]
We will be solving this, but we haven't scheduled anything in, so can't give you a timeframe right now. There are a bunch of form related things that we should be doing very soon: https://github.com/Budibase/budibase/discussions/2033. But I don't think that all of these will be done at once.
